virttest/qemu_devices/qdevices.py

```diff
@@ -2178,9 +2178,9 @@ def _set_device_props(self, device, addr):
         orig_addr = device.get_param('addr')
         if addr[1] or (isinstance(orig_addr, six.string_types) and
                        orig_addr.find('.') != -1):
-            device.set_param('addr', '0x%x.%x' % (addr[0], addr[1]))
+            device.set_param('addr', '%s.%s' % (hex(addr[0]), hex(addr[1])))
         else:
-            device.set_param('addr', '0x%x' % addr[0])
+            device.set_param('addr', '%s' % hex(addr[0]))
```

Comment:
This is not related to the topology change, would you please extract it as a separate commit? (so we can easily revert/modify it in case it breaks something?)

Reply:
Ok, I will separate it a new commit.

Reply:
fix in new commit.